src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -35,6 +35,14 @@ Pane::Pane(const GUID& profile, const TermControl& control, const bool lastFocus
     _border.Child(_control);
 
     _connectionStateChangedToken = _control.ConnectionStateChanged({ this, &Pane::_ControlConnectionStateChangedHandler });
+    _fontSizeChangedToken = _control.FontSizeChanged({ this, &Pane::_FontSizeChangedHandler });
+
+    if (isRoot)
+    {
+        ShouldRelayout([=] {
+            ResizeContent(_root.ActualSize());
+        });
+    }

```

Comment:
I regards to all this `ShouldRelayout` bits:
I think it might be easiest if the `Tab` attached an event handler in `Tab::_AttachEventHandlersToControl` to the control's `FontSizeChanged` event. That way, each Pane wouldn't need to bubble that event up to the root, and none of them would need to know whether they were the root or not. 
```c++
    control.FontSizeChanged([this](const int /* fontWidth */,
                                   const int /* fontHeight */,
                                   const bool isInitialChange) {
        if (isInitialChange)
        {
            _rootPane->Relayout();
        }
    });
```
Where `Pane::Relayout` is just `ResizeContent(_root.ActualSize());`

Then, we wouldn't need the `ShouldRelayout` event at all

Reply:
Yes, that's simpler and cleaner, except only that it's rather a pane's and not tab's thing to care about that. But I'll go with it.